src/aks-preview/azext_aks_preview/custom.py

```diff
@@ -420,6 +423,15 @@ def aks_create(cmd, client, resource_group_name, name, ssh_key_value,  # pylint:
             public_keys=[ContainerServiceSshPublicKey(key_data=ssh_key_value)])
         linux_profile = ContainerServiceLinuxProfile(admin_username=admin_username, ssh=ssh_config)
 
+    windows_profile = None
+    if (windows_admin_username is None) != (windows_admin_password is None):
+        raise CLIError("Please make sure both windows admin username and password are specified.")
```

Comment:
The idiomatic error here would be:
`usage error: --windows-admin-username NAME --windows-password PASSWORD`

Does the command allow secure prompting for the password?

Reply:
@tjprescott Added secure prompt for windows password. Please take a look.